src/redux/sagas/motions/createEditDomainMotion.ts

```diff
@@ -35,10 +33,175 @@ import {
   uploadAnnotation,
   initiateTransaction,
   createActionMetadataInDB,
+  getPermissionProofsLocal,
+  getChildIndexLocal,
 } from '../utils/index.ts';
 
 // import { ipfsUpload } from '../ipfs';
 
+type CreateTransactionPayloadBase = {
```

Comment:
I don't think we actually need such verbose typings with splitting `base` and `motion` parts of the payload.
We can just use a single type for the motion's payload and use that.
Base and motion variants would be useful for reusing types between the permissions action and this motion, currently it's just splitting a payload into 2 smaller objects.

Reply:
In its previous iteration, there used to be a separate and specific payload set for reputation motions and multi-sig. I shall clean this up!